src/config/firebase.js

```diff
@@ -0,0 +1,23 @@
+import { initializeApp } from 'firebase/app';
+import { getFirestore } from 'firebase/firestore';
+import { getRemoteConfig, fetchAndActivate } from 'firebase/remote-config';
+
+const firebaseConfig = {
+  apiKey: 'AIzaSyBp9a0BfdOrwqzgVUFQHrFB7z-1oWYJiOs',
+  authDomain: 'react-firestore-exercise.firebaseapp.com',
+  projectId: 'react-firestore-exercise-4ae95',
+  appId: '491623188913',
```

Comment:
Don't upload these keys on github, add them in an env file in this way https://vite.dev/guide/env-and-mode